### PR TITLE
Add digest_run_subscriber_id index

### PR DESCRIPTION
**Pull request by gpeng**
We don't join on this currently within the app but it's helpful in some diagnostic queries.

### CODE REVIEW

File: db/migrate/20180216100846_add_subscription_contents_index.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddSubscriptionContentsIndex < ActiveRecord::Migration[5.1]
+  def change
+    add_index :subscription_contents, :digest_run_subscriber_id
```
Comment (thomasleese) [inline db/migrate/20180216100846_add_subscription_contents_index.rb]: Do you know how long this takes and if we have to add the index concurrently?

Comment (gpeng) [reply db/migrate/20180216100846_add_subscription_contents_index.rb]: No... But I do have a DB backup now so I'll find out.

Comment (gpeng) [reply db/migrate/20180216100846_add_subscription_contents_index.rb]: It was 0.1218s on my machine with 262751 `SubscriptionContent` records.

Comment (thomasleese) [reply db/migrate/20180216100846_add_subscription_contents_index.rb]: That seems fine to me.

**Pull request merged**